### DISCUSSION TITLE
Treeview screenshot

### DISCUSSION THREAD

**Discussion by GHExplorer** [tui-cs/Terminal.Gui#1373] [Q&A]
Hi,

Is there a screen shot example of the Treeview component anywhere?

**Comment by Ranger8de** [comment]
On the main Page of this repo, there's a nice GIF, witch shows (i think)  every component. Just watch and enjoy 😅

**Reply by GHExplorer** [reply]
Yep, there it is. Ok, thanks Ranger8d.

**Comment by tznind** [comment]
Here you go? you can run UICatalog test project in this repo if you want to experiment with the controls

![image](https://user-images.githubusercontent.com/31306100/126028508-4bae3eae-3cfc-4b1c-ae85-45ad11b7d7a7.png)

**Reply by GHExplorer** [reply]
Looks good, thanks tznind.
